webapp-django/crashstats/supersearch/jinja2/supersearch/search_results.html

```diff
@@ -30,35 +31,19 @@
                   {% if column == 'date' %}
                     {{ crash[column] | human_readable_iso_date }}
                   {% elif column == 'signature' %}
-                    <a href="{{ url('signature:signature_report') }}{{ change_query_string(
-                       _no_base=True,
-                       signature=crash.signature,
-                       page=None,
-                       _columns=None,
-                       _facets=None
-                       ) }}"
-                       title="Click to leave this search and go to this signature report"
-                    >
-                      {{ crash[column] }}
-                    </a>
+                    {{ signature_link(column, crash[column]) }}
                     &nbsp;
-                    <a href="{{ current_url }}&amp;{{ column }}=%3D{{ crash[column] | urlencode }}" class="term" data-field="{{ column }}" data-content="{{ crash[column] }}" title="Add this term to the search form">
-                      Add term
-                    </a>
+                    {{ refine_link(column, crash[column], "Add term") }}
                   {% elif is_list(crash[column]) %}
                     {% for val in crash[column] %}
-                      <a href="{{ current_url }}&amp;{{ column }}=%3D{{ val | urlencode }}" class="term" data-field="{{ column }}" data-content="{{ val }}" title="Add this term to the search form">
-                        {{ val }}
-                      </a>
+                      {{ refine_link(column, val) }}
                     {% endfor %}
+                  {% elif column in ('product', 'version', 'platform', 'process_type') %}
+                    {{ refine_link(column, crash[column], use_is_query=False) }}
```

Comment:
I added this section because these fields are special. `product` and `version` also affect site navigation and other things and the value has to be a full value and can't start with `=`. `platform` and `process_type` go in the supersearch form and if the values start with `=`, then that doesn't work right.